lib/widgets/select_color.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+class SelectColor extends StatefulWidget {
```

Comment:
![image](https://user-images.githubusercontent.com/72055285/116955981-ae238e80-acc6-11eb-85ff-2c54bf815cea.png)
- overflow, is this the right size?

Reply:
for the select color only

Reply:
I used the dimensions in figma (414x896) and they should fit perfectly, for phones with thinner width, do you want the color buttons to be smaller squares or thin rectangles?

Reply:
not sure to be honest but it might look better if we use smaller squares.

let's fix stuff like this probably next week when we have the complete qpi screens